install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -168,56 +321,89 @@ To enable writing to NFS volumes with SELinux enforcing on each node, run:
 # setsebool -P virt_use_nfs 1
```

Comment:
@swagiaal which of these do we need?  I'm positive people in the wild have needed both.